### PR TITLE
Migrate critical constants and acentric factors batch 2

### PR DESCRIPTION
**Pull request by saikat-samanta**
Hey @manufac-analytics/devs , few no. of compound name started with number. Please check them🙏

### CODE REVIEW

File: src/data/critical-constants-acentric-factor.ts [hunk 1]
```diff
@@ -15,4 +15,616 @@ export const DIPPRCriticalConstantsAndAcentricFactorDictionary: Record<
...
+    CAS: '74-83-9',
+    molecularWeight: 94.93852,
+    criticalTemperature: 464,
+    criticalPressure: 6.929,
+    criticalVolume: 0.152,
+    criticalCompressibilityFactor: 0.273,
+    acentricFactor: 0.153426,
+  },
+
+  Butadiene: {
```
Comment (qur786) [inline src/data/critical-constants-acentric-factor.ts:343]:
```suggestion
  Butadiene_1_2: {
```

**Review by qur786 — CHANGES_REQUESTED** [review]

File: src/data/critical-constants-acentric-factor.ts [hunk 2]
```diff
@@ -15,4 +15,616 @@ export const DIPPRCriticalConstantsAndAcentricFactorDictionary: Record<
...
+    CAS: '590-18-1',
+    molecularWeight: 56.10632,
+    criticalTemperature: 435.5,
+    criticalPressure: 4.21,
+    criticalVolume: 0.234,
+    criticalCompressibilityFactor: 0.272,
+    acentricFactor: 0.201877,
+  },
+
+  trans_2_Butene: {
```
Comment (sumitnegi7) [inline src/data/critical-constants-acentric-factor.ts:451]:
```suggestion
  trans2Butene: {
```

File: src/data/critical-constants-acentric-factor.ts [hunk 3]
```diff
@@ -15,4 +15,616 @@ export const DIPPRCriticalConstantsAndAcentricFactorDictionary: Record<
...
+    CAS: '106-98-9',
+    molecularWeight: 56.10632,
+    criticalTemperature: 419.5,
+    criticalPressure: 4.02,
+    criticalVolume: 0.241,
+    criticalCompressibilityFactor: 0.278,
+    acentricFactor: 0.184495,
+  },
+
+  cis_2_Butene: {
```
Comment (sumitnegi7) [inline src/data/critical-constants-acentric-factor.ts:439]: cis2Butene

File: src/data/critical-constants-acentric-factor.ts (hunk 1 above)
Comment (maneetgoyal) [inline src/data/critical-constants-acentric-factor.ts:343]:
```suggestion
  '1,2Butadiene': {
```

Don't switch the order of the prefixes.

File: src/data/critical-constants-acentric-factor.ts [hunk 4]
```diff
@@ -15,4 +15,616 @@ export const DIPPRCriticalConstantsAndAcentricFactorDictionary: Record<
...
+    CAS: '590-19-2',
+    molecularWeight: 54.09044,
+    criticalTemperature: 452,
+    criticalPressure: 4.36,
+    criticalVolume: 0.22,
+    criticalCompressibilityFactor: 0.255,
+    acentricFactor: 0.165877,
+  },
+
+  Butadiene_1_3: {
```
Comment (maneetgoyal) [inline src/data/critical-constants-acentric-factor.ts:355]:
```suggestion
  '1,3Butadiene': {
```

File: src/data/critical-constants-acentric-factor.ts [hunk 5]
```diff
@@ -15,4 +15,616 @@ export const DIPPRCriticalConstantsAndAcentricFactorDictionary: Record<
...
+    CAS: '106-97-8',
+    molecularWeight: 58.1222,
+    criticalTemperature: 425.12,
+    criticalPressure: 3.796,
+    criticalVolume: 0.255,
+    criticalCompressibilityFactor: 0.274,
+    acentricFactor: 0.200164,
+  },
+
+  Butanediol_1_2: {
```
Comment (maneetgoyal) [inline src/data/critical-constants-acentric-factor.ts:379]:
```suggestion
  '1,2Butanediol': {
```

File: src/data/critical-constants-acentric-factor.ts [hunk 6]
```diff
@@ -15,4 +15,616 @@ export const DIPPRCriticalConstantsAndAcentricFactorDictionary: Record<
...
+    name: ' 1,2-Butanediol',
```
Comment (maneetgoyal) [inline src/data/critical-constants-acentric-factor.ts:380]:
```suggestion
    name: '1,2-Butanediol',
```

File: src/data/critical-constants-acentric-factor.ts (hunk 5 above)
Comment (maneetgoyal) [reply src/data/critical-constants-acentric-factor.ts:379]: and so on.

**Comment by saikat-samanta** [comment]
@manufac-analytics/devs, Requested changes are done 👍

**Review by maneetgoyal — APPROVED** [review]

**Review by qur786 — APPROVED** [review]
all good

**Pull request merged**